src/googleclouddebugger/firebase_client.py

```diff
@@ -343,6 +339,40 @@ def _StartMarkActiveTimer(self):
                                               self._MarkActiveTimerFunc)
     self._mark_active_timer.start()
 
+  def _ConnectToDb(self):
+    urls = [self._database_url] if self._database_url is not None else \
+      [f'https://{self._project_id}-cdbg.firebaseio.com',
+      f'https://{self._project_id}-default-rtdb.firebaseio.com']
+
+    for url in urls:
+      native.LogInfo(f'Attempting to connect to DB with url: {url}')
+      if self._TryInitializeDbForUrl(url):
+        native.LogInfo(f'Successfully connected to DB with url: {url}')
+        self._database_url = url
+        return (False, 0)  # Proceed immediately to registering the debuggee.
+
+    return (True, self.register_backoff.Failed())
+
+  def _TryInitializeDbForUrl(self, database_url):
+    # Note: if self._credentials is None, default app credentials will be used.
+    app = None
+    try:
+      app = firebase_admin.initialize_app(self._credentials,
```

Comment:
While we're in this code, it would be good to name the app to reduce the likelihood of conflicts.